pkg/server/server.go

```diff
@@ -1618,24 +1625,29 @@ func (s *Server) checkCreateStoreAuthz(ctx context.Context) error {
 // getAccessibleStores checks whether the caller has permission to list stores and if so,
 // returns the list of stores that the user has access to.
 func (s *Server) getAccessibleStores(ctx context.Context) ([]string, error) {
-	if s.authorizer != nil {
-		err := s.authorizer.AuthorizeListStores(ctx)
-		if err != nil {
-			return nil, err
-		}
+	if authclaims.SkipAuthzCheckFromContext(ctx) {
+		return nil, nil
+	}
 
-		list, err := s.authorizer.ListAuthorizedStores(ctx)
-		if err != nil {
-			return nil, err
-		}
+	err := s.authorizer.AuthorizeListStores(ctx)
```

Comment:
I'm assuming we removed the `if s.authorizer != nil` b/c it can never be nil now right? It's noop by default?

Reply:
Yup there's a noop function for both of these where it just returns nil